react/features/base/participants/reducer.ts

```diff
@@ -114,6 +115,20 @@ export interface IParticipantsState {
 ReducerRegistry.register<IParticipantsState>('features/base/participants',
 (state = DEFAULT_STATE, action): IParticipantsState => {
     switch (action.type) {
+    case NOTIFIED_TO_SPEAK: {
+        return {
+            ...state,
+            raisedHandsQueue: state.raisedHandsQueue.map((item, index) => {
+                if (index === 0) {
+                    return { ...item,
```

Comment:
You don't need to clone the item entirely, since we only depend on the queue being unique, but it won't hurt either.